config/routes.rb

```diff
@@ -14,6 +14,7 @@
 
   get '/content', to: 'content#show'
   get '/single_page/*base_path', to: 'single_item#show', defaults: { format: :json }
```

Comment:
How about changing this to `get '/single_page*base_path'`, would that work?

All the base_path's in the Publishing API start with `/`, so this would be more consistent generally.

Reply:
It didn't work but now I've found a better solution.